frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <v-skeleton-loader :loading="loading" type="table-tbody">
+    <v-card class="my-4 pa-4 w-100" variant="outlined">
+      <v-container fluid class="pa-0">
+        <AppLabel>Not for Profit Information</AppLabel>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4">
+          <v-row no-gutters>
+            <v-col>
+              <AppLabel>Date of Incorporation</AppLabel>
+            </v-col>
+          </v-row>
+          <AppDateInput id="date-to" v-model="model.dateOfIncorporation" :rules="[...rules.required, rules.MMDDYYYY]" :max="todayDate" :disabled="false" label="Select The Date" class="mt-6 mx-6" />
+        </v-card>
+        <v-card v-if="isReferenceLetterRequired" variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col cols="7" md="3" lg="2">
+              <AppLabel>Upload Documents</AppLabel>
+            </v-col>
```

Comment:
as discussed, we can remove this v-col so there'll be no gap between the label and the tooltip (similar to how we do in LicenceDetails.vue)

Reply:
done